trio/_subprocess.py

```diff
@@ -55,22 +56,19 @@ def pidfd_open(fd, flags):
 class Process(AsyncResource, metaclass=NoPublicConstructor):
     r"""A child process. Like :class:`subprocess.Popen`, but async.
 
-    This class has no public constructor. To create a child process, use
-    `open_process`::
+    This class has no public constructor. The most common way to create a
+    `Process` is to combine `Nursery.start` with `run_process`::
 
-       process = await trio.open_process(...)
+       process = await nursery.start(run_process, ...)
 
-    `Process` implements the `~trio.abc.AsyncResource` interface. In order to
-    make sure your process doesn't end up getting abandoned by mistake or
-    after an exception, you can use ``async with``::
+    This way, `run_process` supervises the process, and makes sure that it is
+    cleaned up, while optionally checking the output, feeding it input, and so
+    on.
 
-       async with await trio.open_process(...) as process:
-           ...
+    If you need more control – for example, because you want to spawn a child
+    process that outlives your program – then you can use `open_process`::
```

Comment:
Wait is this true? 

Can `open_process()` outlive the "program" given the `subprocess` routine is launched using [.to_thread.run()`](https://github.com/python-trio/trio/blob/c0dbe792095b67655cb14deee42d4a050625846b/trio/_subprocess.py#L369)?

Maybe "program" isn't the ideal term here?

Reply:
We use a thread to spin up the process, but once it's started it's an independent entity.

Reply:
Right, my quiff is on the use of the term *program*.
I presume no thread outlives the `trio` program given the `deamon=True` flag is used iirc?

Reply:
`open_process` doesn't spawn a thread, it spawns a process... there are no threads here (except very briefly as an implementation detail that isn't mentioned in the docs)

Reply:
Ahh i see. I didn't realize you could intentionally leak processes; gtk.